tools/Performance/DynamoPerformanceTests/PerformanceTestFramework.cs

```diff
@@ -44,7 +44,12 @@ public BenchmarkConfig(string testDir)
             {
                 Add(DefaultConfig.Instance.GetLoggers().ToArray()); // manual config has no loggers by default
                 Add(DefaultConfig.Instance.GetExporters().ToArray()); // manual config has no exporters by default
-                Add(DefaultConfig.Instance.GetColumnProviders().ToArray());
+
+                var defaultColumns = DefaultConfig.Instance.GetColumnProviders().ToList();
+                defaultColumns.RemoveAt(3); // Remove DynamoFilePath column
```

Comment:
@scottmitchell Is this column added by default or added somewhere else in the code? I am wondering if we could skip that part elsewhere, but I could be wrong

Reply:
@QilongTang It automatically gets created when you create the parameter sources [here](https://github.com/DynamoDS/Dynamo/blob/master/tools/Performance/DynamoPerformanceTests/PerformanceTestFramework.cs#L79). I agree that it would be better to skip the creation of this column at all, but I'm not sure that it's possible. I'll look into it further.

Reply:
@aparajit-pratap Do you know if there's a way to suppress the creation of the `DynamoFilePath` column?

Reply:
Nope, I don't. Need to check.

Reply:
@scottmitchell I think we may forgot to apply the same changes here to the other debug config, I will include this in my PR

Reply:
@QilongTang Yeah you're right. Thanks for catching that.